system/nagios-status-check/detectors-nagios-status-check.tf

```
@@ -0,0 +1,27 @@
+resource "signalfx_detector" "status_check" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Custom script status"
```

I think you should add the "Nagios" context in the name of this detector. even if it is fully correct this module and their detectors are dedicated to the nagios logic, maybe : 

```suggestion
  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Nagios check status"
```

---

Done